<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmPosodobiKan</class>
 <widget class="QDialog" name="frmPosodobiKan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>420</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Inter</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Posodobi kanalizacijo</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,5">
   <item>
    <widget class="QFrame" name="frame">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>246</red>
           <green>246</green>
           <blue>246</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>246</red>
           <green>246</green>
           <blue>246</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>246</red>
           <green>246</green>
           <blue>246</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>246</red>
           <green>246</green>
           <blue>246</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>344</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Inter</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Posodobi kanalizacijske podatke v ArcGis Online</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_jaski">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>137</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Kanalizacijski jaški</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_linije">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>90</y>
        <width>143</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Kanalizacijske linije</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_posodobi">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Posodobi</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_cancel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>330</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Nazaj</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_iztok">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>143</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Iztok kanalizacije</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_objekt">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>143</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Kanalizacijski objekt</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_kcn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Čistilna naprava</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_crpalisce">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Črpališče</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_greznica">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Greznica</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_lovilec">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>180</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Lovilec olj</string>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="te2">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Base">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
